homeassistant/components/unifi/device_tracker.py

```diff
@@ -196,6 +191,14 @@ def _make_disconnected(now):
                 self.schedule_update = True
 
         if self.schedule_update:
+
+            @callback
+            def _make_disconnected(_):
```

Comment:
You could make it a little faster by moving this to be an instance method instead of defining it inline.

Reply:
so much cleaner 😄